internal/controller/rest/controller_test.go

```diff
@@ -274,11 +271,9 @@ func TestAddSecretRequest(t *testing.T) {
 		ErrorExpected      bool
 		ExpectedStatusCode int
 	}{
-		{"Valid - sub-path no trailing slash, SecretsPath has trailing slash", validRequest, expectedRequestId, "my-secrets/", "true", false, http.StatusCreated},
-		{"Valid - sub-path only with trailing slash", validPathWithSlash, expectedRequestId, "my-secrets", "true", false, http.StatusCreated},
-		{"Valid - both trailing slashes", validPathWithSlash, expectedRequestId, "my-secrets/", "true", false, http.StatusCreated},
+		{"Valid - sub-path no trailing slash, SecretsPath has trailing slash", validRequest, expectedRequestId, "my-secrets", "true", false, http.StatusCreated},
```

Comment:
Still have the reference to slashes..
```suggestion
		{"Valid name", validRequest, expectedRequestId, "my-secrets", "true", false, http.StatusCreated},
```

Reply:
Also the  `SecretsPath` in the test struct above need to be `SecretName`. Actually In don't see it used anywhere, so it should be removed.